{{ cookiecutter.__project_name_kebab }}/.nvmrc

```
@@ -1 +1 @@
-{{ cookiecutter.node_version }}
+18
```

```suggestion
20
```

See https://github.com/wagtail/cookiecutter-wagtail-package/issues/69